.github/workflows/release.yaml

```diff
@@ -97,7 +103,23 @@ jobs:
             }
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          
+      
+      - name: Create Git tag for api submodule
+        if: ${{ env.SKIP != 'true' }}
+        run: |
+          AUTHOR_NAME=$(git log -1 --pretty=format:'%an')
+          AUTHOR_EMAIL=$(git log -1 --pretty=format:'%ae')
+          echo "Tagging as $AUTHOR_NAME <$AUTHOR_EMAIL>"
+ 
+          echo "AUTHOR_NAME=$AUTHOR_NAME" >> $GITHUB_ENV
+          echo "AUTHOR_EMAIL=$AUTHOR_EMAIL" >> $GITHUB_ENV
+ 
+          git config user.name "$AUTHOR_NAME"
+          git config user.email "$AUTHOR_EMAIL"
+ 
+          git tag -a "api/${{ env.version }}" -m "Release ${{ env.version }}"
+          git push origin "api/${{ env.version }}"
+
```

Comment:
This repo's `api` folder is not a nested go module (it does not contain a `go.mod` file), so we don't need this part.
```suggestion
```